javascript/javascript1/week1/Goodboy-Oldboy.js

```diff
@@ -3,6 +3,7 @@ let dogYearFuture = 2027;
 let dogYear = dogYearFuture - dogYearOfBirth;
 //console.log(dogYear);
```

Comment:
remove the commented code while commit the code unless it is description or for the future understanding.

Reply:
okay kavya thank you.